.changeset/little-fans-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/next-drupal-starter": patch
```

Comment:
Originally went with patch because new features were not added but thinking about this more I say major since these updates are not backwards compatible and patch/minor imply backwards compatibility.